app/components/map/legend/types/gradient/component.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cx from 'classnames';
+
+export interface LegendTypeGradientProps {
+  className?: string;
+  items: Array<{
+    value: string;
+    color: string;
+  }>;
+}
```

Comment:
Any reason why legends cannot share the same interface?

Reply:
It depends on the type. It's true that now they have the same interface, but I could imagine basic type having their own items. For example, you could define the icon for basic ones. That's why I want to keep them separate.

Reply:
@mbarrenechea sure thing, they can just extend base later on - but they still have the same base ; ) Even tho they have different name, TS consider them to be exactly the same.